src/components/ContributionsEpic.tsx

```diff
@@ -226,6 +228,34 @@ export const ContributionsEpic: React.FC<Props> = ({
                     className={paymentImageStyles}
                 />
             </div>
+
+            {showReminderFields && (
+                <ContributionsEpicReminder
+                    reminderDate={showReminderFields.reminderDate}
+                    reminderDateAsString={showReminderFields.reminderDateAsString}
+                />
+            )}
         </section>
     );
 };
+
+export interface SlotComponent {
+    component: JSX.Element;
+    js: string;
+}
+
+export const contributionsEpicSlot = (props: Props): SlotComponent => {
+    let js = '';
+    if (props.variant.showReminderFields) {
+        const contributionsReminderUrl =
+            process.env.NODE_ENV === 'production'
```

Comment:
Sorry I was late to notice this - I wonder if specifying the URL in an env var directly, rather than deriving on `NODE_ENV` would make sense? Something like `CONTRIBUTIONS_REMINDER_URL`? Whatever you think, totally fine to leave as-is in this PR.

Reply:
As discussed on Chat, we're leaving this as is, potentially moving all the URLs to env variables later on in one go.